specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -0,0 +1,1524 @@
+          }
+        }
+      }
+    },
+    "/prebuilt/receipt/analyze": {
+      "post": {
+        "summary": "Analyze Receipt",
+        "description": "Extract field text and semantic values from a given receipt document.",
+        "operationId": "AnalyzeReceiptAsync",
+        "consumes": [
+          "application/pdf",
+          "application/json",
+          "image/jpeg",
+          "image/png",
+          "image/tiff"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/FileStream"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Request to analyze receipt queued successfully.",
+            "headers": {
+              "Location": {
+                "type": "string",
+                "description": "Operation location with an operation id used to track the progress of analyze operation. The returned id is the operation id. Result of analyze operation can be obtained only when the operation status becomes \"succeeded\" in OperationStatus - Get."
+              }
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Analyze receipt": {
+            "$ref": "./examples/ReceiptsBatch.json"
+          }
+        }
+      }
+    },
+    "/prebuilt/receipt/analyzeResults/{resultId}": {
+      "get": {
+        "summary": "Get Receipt Result",
+        "description": "Retrieve analyze receipt operation result.",
+        "operationId": "GetAnalyzeReceiptOpResult",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "resultId",
+            "in": "path",
+            "description": "Analyze operation identifier.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeOperationResult"
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Analyze receipt": {
+            "$ref": "./examples/ReceiptsBatchResult.json"
+          }
+        }
+      }
+    }
+  },  
+  "definitions": {
+    "OperationStatus": {
+      "type": "string",
+      "description": "Status of queued operation.",
+      "enum": [
+        "notStarted",
```

Comment:
Should we unify this enum with status of train op?

Reply:
If we did, we will end up with model.status == succeeded vs. model.status=ready, not sure if that coveys correct model status.

Reply:
We need more discussion on status. Train status and model status need to be unified somehow.